### PR TITLE
TestNG implementation

### PR DESCRIPTION
**Pull request by iuriestanila**

### CODE REVIEW

File: pom.xml [hunk 1]
```diff
@@ -13,8 +13,8 @@
                 <groupId>org.apache.maven.plugins</groupId>
                 <artifactId>maven-compiler-plugin</artifactId>
                 <configuration>
-                    <source>1.7</source>
-                    <target>1.7</target>
+                    <source>8</source>
```
Comment (shukal94) [inline pom.xml]: update to 17

File: src/test/java/TestNGTests/CartTest.java [hunk 2]
```diff
@@ -0,0 +1,54 @@
...
+        car.setWeight(1560);
+        andrewCart.addRealItem(car);
+
+        final double expected = andrewCart.getTotalPrice();
+        softAssert.assertEquals(expected, andrewCart.getTotalPrice());
+        softAssert.assertAll();
+    }
+
+    @Parameters({"name", "price","weight"})
+    @Test(groups = "regression", priority = 1)
```
Comment (shukal94) [inline src/test/java/TestNGTests/CartTest.java]: remove all the priorities, orders and depends

**Review by shukal94 — CHANGES_REQUESTED** [review]
1) let's move the suites into src/test/resources/testng_sutes
2) package names should be in lower case, e.g. `parser` instead of `TestNGTests`

File: src/test/java/TestNGTests/CartTest.java (hunk 2 above)
Comment (iuriestanila) [reply src/test/java/TestNGTests/CartTest.java]: I have removed priorities. Orders I quess are invocationCount (removed). At least I don't see any dependents in my code. I know that this is a dependency - dependsOnMethods. Or there are other established dependenies which I do not see.

**Review by shukal94 — APPROVED** [review]

**Pull request merged**